diff -up a/numpy/core/setup.py.0000~ a/numpy/core/setup.py
--- a/numpy/core/setup.py.0000~	2019-02-26 17:42:42.000000000 +0100
+++ a/numpy/core/setup.py	2019-04-13 14:54:06.614427058 +0200
@@ -930,7 +930,7 @@ def configuration(parent_package='',top_
                                  ],
                          depends=deps + multiarray_deps + umath_deps +
                                 common_deps,
-                         libraries=['npymath', 'npysort'],
+                         libraries=['m', 'npymath', 'npysort'],
                          extra_info=extra_info)
 
     #######################################################################
@@ -938,7 +938,7 @@ def configuration(parent_package='',top_
     #######################################################################
 
     config.add_extension('_umath_tests',
-                    sources=[join('src', 'umath', '_umath_tests.c.src')])
+                    sources=[join('src', 'umath', '_umath_tests.c.src')],libraries=['m'])
 
     #######################################################################
     #                   custom rational dtype module                      #
diff -up a/numpy/fft/setup.py.0000~ a/numpy/fft/setup.py
--- a/numpy/fft/setup.py.0000~	2019-02-26 17:42:42.000000000 +0100
+++ a/numpy/fft/setup.py	2019-04-13 14:52:46.224226669 +0200
@@ -9,7 +9,8 @@ def configuration(parent_package='',top_
 
     # Configure fftpack_lite
     config.add_extension('fftpack_lite',
-                         sources=['fftpack_litemodule.c', 'fftpack.c']
+                         sources=['fftpack_litemodule.c', 'fftpack.c'],
+                         libraries=['m']
                          )
 
     return config
diff -up a/numpy/random/setup.py.0000~ a/numpy/random/setup.py
--- a/numpy/random/setup.py.0000~	2019-04-13 14:52:46.224226669 +0200
+++ a/numpy/random/setup.py	2019-04-13 14:54:28.496900510 +0200
@@ -39,7 +39,7 @@ def configuration(parent_package='',top_
     if needs_mingw_ftime_workaround():
         defs.append(("NPY_NEEDS_MINGW_TIME_WORKAROUND", None))
 
-    libs = []
+    libs = ['m']
     # Configure mtrand
     config.add_extension('mtrand',
                          sources=[join('mtrand', x) for x in
