--- python-numpy-1.14.0/numpy/core/setup.py.omv~	2018-02-09 19:07:14.587881073 +0100
+++ python-numpy-1.14.0/numpy/core/setup.py	2018-02-09 19:07:43.944980567 +0100
@@ -693,6 +693,7 @@ def configuration(parent_package='',top_
             sources=npymath_sources + [get_mathlib_info],
             install_dir='lib',
             build_info={
+                'libraries': [ 'm' ],
                 'include_dirs' : [],  # empty list required for creating npy_math_internal.h
                 'extra_compiler_args' : (['/GL-'] if is_msvc else []),
             })
