diff --git a/numpy/core/setup.py b/numpy/core/setup.py
index f826b27..39d5775 100644
--- a/numpy/core/setup.py
+++ b/numpy/core/setup.py
@@ -909,7 +909,7 @@ def configuration(parent_package='',top_path=None):
                                  generate_umath_c,
                                  generate_ufunc_api],
                          depends=deps + umath_deps,
-                         libraries=['npymath'],
+                         libraries=['m','npymath'],
                          )
 
     #######################################################################
@@ -917,7 +917,7 @@ def configuration(parent_package='',top_path=None):
     #######################################################################
 
     config.add_extension('_umath_tests',
-                    sources=[join('src', 'umath', '_umath_tests.c.src')])
+                    sources=[join('src', 'umath', '_umath_tests.c.src')],libraries=['m'])
 
     #######################################################################
     #                   custom rational dtype module                      #
diff --git a/numpy/fft/setup.py b/numpy/fft/setup.py
index cd99a82..e614ecd 100644
--- a/numpy/fft/setup.py
+++ b/numpy/fft/setup.py
@@ -9,7 +9,8 @@ def configuration(parent_package='',top_path=None):
 
     # Configure fftpack_lite
     config.add_extension('fftpack_lite',
-                         sources=['fftpack_litemodule.c', 'fftpack.c']
+                         sources=['fftpack_litemodule.c', 'fftpack.c'],
+                         libraries=['m']
                          )
 
     return config
diff --git a/numpy/random/setup.py b/numpy/random/setup.py
index a8d82b1..6f7d9e3 100644
--- a/numpy/random/setup.py
+++ b/numpy/random/setup.py
@@ -43,7 +43,7 @@ def configuration(parent_package='',top_path=None):
     # see https://github.com/cython/cython/issues/2494
     defs.append(('CYTHON_SMALL_CODE', ''))
 
-    libs = []
+    libs = ['m']
     # Configure mtrand
     config.add_extension('mtrand',
                          sources=[join('mtrand', x) for x in
